sdk/storage/storage-blob/test/blobclientpollers.spec.ts

```diff
@@ -22,32 +23,37 @@ describe("BlobClient beginCopyFromURL Poller", () => {
   let recorder: Recorder;
 
   beforeEach(async function (this: Context) {
-    recorder = record(this, recorderEnvSetup);
-    const blobServiceClient = getBSU();
-    containerName = recorder.getUniqueName("container");
+    recorder = new Recorder(this.currentTest);
+    await recorder.start(recorderEnvSetup);
+    const blobServiceClient = getBSU(recorder);
+    containerName = recorder.variable("container", getUniqueName("container"));
     containerClient = blobServiceClient.getContainerClient(containerName);
     await containerClient.create();
-    blobName = recorder.getUniqueName("blob");
+    blobName = recorder.variable("blob", getUniqueName("blob"));
     blobClient = containerClient.getBlobClient(blobName);
     blockBlobClient = blobClient.getBlockBlobClient();
     await blockBlobClient.upload(content, content.length);
-    destinationContainerName = recorder.getUniqueName("dest-container");
+    destinationContainerName = recorder.variable("dest-container", getUniqueName("dest-container"));
     destinationContainerClient = blobServiceClient.getContainerClient(destinationContainerName);
     await destinationContainerClient.create();
   });
 
   afterEach(async function (this: Context) {
-    if (!this.currentTest?.isPending()) {
+    if (containerClient) {
       await containerClient.delete();
+    }
+    if (destinationContainerClient) {
       await destinationContainerClient.delete();
-      await recorder.stop();
     }
+    await recorder.stop();
   });
 
-  it("supports automatic polling via pollUntilDone", async () => {
-    recorder.skip("browser");
+  it("supports automatic polling via pollUntilDone", async function () {
+    if (!isNode && !isLiveMode()) {
+      this.skip();
+    }
```

Comment:
Is it being skipped in non-live modes too?

Reply:
I think it's only skipped in playback/record when it's node?

Reply:
misread, matches with what was there before